<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskDisplayDialog</class>
 <widget class="QDialog" name="TaskDisplayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>展示任务</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    /* 基础样式 */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0.2 #f0f8ff, stop:1 #d3e9fd); /* 清新的浅蓝色渐变背景 */
    border: 2px solid #4a90e2; /* 深蓝色边框 */
    border-radius: 12px; /* 圆角 */
    padding: 20px;
    min-width: 300px;
    min-height: 200px;
    max-width: 600px;
    max-height: 500px;
    box-shadow: 0 4px 10px rgba(74, 144, 226, 0.2); /* 轻微阴影效果 */
}

/* 标题栏样式 */
QDialog QMenuBar {
    background: #4a90e2; /* 蓝色标题栏背景 */
    color: #ffffff; /* 白色文字 */
    border: none;
    height: 35px;
}

QDialog QMenuBar::item {
    padding: 5px 15px;
}

QDialog QMenuBar::item:selected {
    background-color: #82aef2; /* 选中时的背景色 */
}

/* 按钮（如OK/Cancel）样式 */
QDialog QPushButton {
    background: #4a90e2; /* 按钮背景颜色 */
    color: white; /* 白色文本 */
    border: 1px solid #4a90e2; /* 边框颜色与背景一致 */
    border-radius: 8px;
    padding: 8px 16px;
    font-size: 14px;
    font-weight: bold;
}

QDialog QPushButton:hover {
    background: #82aef2; /* 鼠标悬停时按钮背景色变浅 */
    border-color: #82aef2; /* 悬停时按钮边框颜色变浅 */
}

QDialog QPushButton:pressed {
    background: #1a7bb9; /* 按钮按下时颜色变为深蓝 */
    border-color: #1a7bb9; /* 按钮按下时边框变为深蓝 */
}

/* 文本框样式（如 QLineEdit、QTextEdit） */
QDialog QLineEdit, QDialog QTextEdit {
    background-color: #ffffff; /* 背景为白色 */
    border: 1px solid #d1e8fd; /* 边框为浅蓝色 */
    border-radius: 6px; /* 圆角 */
    padding: 10px;
    font-size: 14px;
    color: #333333; /* 字体颜色为深灰 */
}

QDialog QLineEdit:focus, QDialog QTextEdit:focus {
    border: 1px solid #4a90e2; /* 聚焦时边框变为蓝色 */
    background-color: #e6f1fd; /* 聚焦时背景变为浅蓝 */
}

/* 关闭按钮样式 */
QDialog QPushButton#closeButton {
    background: transparent; /* 透明背景 */
    color: #ff4d4d; /* 关闭按钮的红色 */
    font-size: 18px;
    border: none;
    font-weight: bold;
}

QDialog QPushButton#closeButton:hover {
    color: #ff1a1a; /* 悬停时颜色变得更深 */
}

/* 各种文本样式 */
QDialog QLabel {
    color: #4a90e2; /* 标签文本颜色与对话框的主色调一致 */
    font-size: 16px;
    padding-bottom: 10px;
}

/* 设置 QDialog 中的内容区域 */
QDialog QWidget {
    background: #ffffff; /* 内容区域背景为白色 */
    border-radius: 8px;
    padding: 15px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_main_descriptions">
           <property name="styleSheet">
            <string notr="true">QLabel {
    /* 基础样式 */
    font-family: &quot;Segoe UI&quot;, &quot;Helvetica&quot;, sans-serif;
    font-size: 16px;
    font-weight: bold; /* 设置字体加粗 */
    color: #4a90e2; /* 主文本颜色 - 清新的蓝色 */
    padding: 12px;
    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0.2 #e6f1fd, stop:1 #a9c9f5); /* 清新的蓝色渐变背景 */
    border: 1px solid #4a90e2; /* 蓝色边框 */
    text-align: center;
    min-width: 160px;
    min-height: 40px;
    max-width: 300px;
    max-height: 80px;
}

/* 鼠标悬停效果 */
QLabel:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0.2 #d2e8fb, stop:1 #82aef2); /* 悬停时的渐变 */
    color: #1a7bb9; /* 悬停时文本颜色更深 */
    border: 1px solid #1a7bb9; /* 悬停时边框颜色更深 */
}

/* 点击时的效果 */
QLabel:pressed {
    background-color: #c3dafb; /* 按下时背景更浅 */
    color: #1a7bb9; /* 按下时文本颜色深 */
    border-color: #1a7bb9; /* 按下时边框深色 */
}

/* 禁用状态 */
QLabel:disabled {
    color: #b0c4de; /* 禁用状态的文本颜色 */
    background-color: #f0f4f8; /* 禁用状态的背景色 */
    border: 1px solid #d1dbe6; /* 禁用状态的边框 */
}

/* 聚焦效果 */
QLabel:focus {
    border: 2px solid #0078d4; /* 聚焦时边框颜色变为深蓝色 */
}

/* 如果 QLabel 设置了图片背景 */
QLabel[hasImage=&quot;true&quot;] {
    background: url(:/images/blue_bg.png) no-repeat center center;
    background-size: cover;
    color: #ffffff; /* 背景图片时，文本颜色为白色 */
    padding: 0;
}

/* 自定义标签图标 */
QLabel[icon]:not([hasImage=&quot;true&quot;]) {
    padding-left: 35px;
    background: transparent;
}

QLabel[icon]:not([hasImage=&quot;true&quot;])::before {
    content: url(:/images/icon_blue.png); /* 你可以自定义图标路径 */
    position: absolute;
    left: 12px;
    top: 50%;
    transform: translateY(-50%);
}

/* 更细致的调整：避免字体过大或者过小影响对齐 */
QLabel {
    line-height: 1.4; /* 增加行高，使文本垂直居中 */
    text-align: center;
}

/* 调整带图标时的显示效果 */
QLabel[icon]:not([hasImage=&quot;true&quot;]) {
    padding-left: 45px; /* 增加左侧的空隙，以确保图标不被文本遮挡 */
}
</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_endTime">
           <property name="styleSheet">
            <string notr="true">QLabel {
    /* 基础样式 */
    font-family: &quot;Segoe UI&quot;, &quot;Helvetica&quot;, sans-serif;
    font-size: 16px;
    font-weight: bold; /* 设置字体加粗 */
    color: #4a90e2; /* 主文本颜色 - 清新的蓝色 */
    padding: 12px;
    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0.2 #e6f1fd, stop:1 #a9c9f5); /* 清新的蓝色渐变背景 */
    border: 1px solid #4a90e2; /* 蓝色边框 */
    text-align: center;
    min-width: 160px;
    min-height: 40px;
    max-width: 300px;
    max-height: 80px;
}

/* 鼠标悬停效果 */
QLabel:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0.2 #d2e8fb, stop:1 #82aef2); /* 悬停时的渐变 */
    color: #1a7bb9; /* 悬停时文本颜色更深 */
    border: 1px solid #1a7bb9; /* 悬停时边框颜色更深 */
}

/* 点击时的效果 */
QLabel:pressed {
    background-color: #c3dafb; /* 按下时背景更浅 */
    color: #1a7bb9; /* 按下时文本颜色深 */
    border-color: #1a7bb9; /* 按下时边框深色 */
}

/* 禁用状态 */
QLabel:disabled {
    color: #b0c4de; /* 禁用状态的文本颜色 */
    background-color: #f0f4f8; /* 禁用状态的背景色 */
    border: 1px solid #d1dbe6; /* 禁用状态的边框 */
}

/* 聚焦效果 */
QLabel:focus {
    border: 2px solid #0078d4; /* 聚焦时边框颜色变为深蓝色 */
}

/* 如果 QLabel 设置了图片背景 */
QLabel[hasImage=&quot;true&quot;] {
    background: url(:/images/blue_bg.png) no-repeat center center;
    background-size: cover;
    color: #ffffff; /* 背景图片时，文本颜色为白色 */
    padding: 0;
}

/* 自定义标签图标 */
QLabel[icon]:not([hasImage=&quot;true&quot;]) {
    padding-left: 35px;
    background: transparent;
}

QLabel[icon]:not([hasImage=&quot;true&quot;])::before {
    content: url(:/images/icon_blue.png); /* 你可以自定义图标路径 */
    position: absolute;
    left: 12px;
    top: 50%;
    transform: translateY(-50%);
}

/* 更细致的调整：避免字体过大或者过小影响对齐 */
QLabel {
    line-height: 1.4; /* 增加行高，使文本垂直居中 */
    text-align: center;
}

/* 调整带图标时的显示效果 */
QLabel[icon]:not([hasImage=&quot;true&quot;]) {
    padding-left: 45px; /* 增加左侧的空隙，以确保图标不被文本遮挡 */
}
</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="styleSheet">
            <string notr="true">QProgressBar {
    /* 基本样式 */
    text-align: center; /* 文本居中 */
    background-color: #DDDDDD; /* 背景颜色 */
    border: none; /* 去除边框 */
    border-radius: 5px; /* 圆角 */
}

QProgressBar::chunk {
    /* 进度块样式 */
    background-color: #05B8CC; /* 进度块颜色为深蓝绿色 */
    border-radius: 5px; /* 进度块圆角 */
}
</string>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_time_remain">
           <property name="styleSheet">
            <string notr="true">QLabel {
    /* 基础样式 */
    font-family: &quot;Segoe UI&quot;, &quot;Helvetica&quot;, sans-serif;
    font-size: 16px;
    font-weight: bold; /* 设置字体加粗 */
    color: #4a90e2; /* 主文本颜色 - 清新的蓝色 */
    padding: 12px;
    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0.2 #e6f1fd, stop:1 #a9c9f5); /* 清新的蓝色渐变背景 */
    border: 1px solid #4a90e2; /* 蓝色边框 */
    text-align: center;
    min-width: 160px;
    min-height: 40px;
    max-width: 300px;
    max-height: 80px;
}

/* 鼠标悬停效果 */
QLabel:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0.2 #d2e8fb, stop:1 #82aef2); /* 悬停时的渐变 */
    color: #1a7bb9; /* 悬停时文本颜色更深 */
    border: 1px solid #1a7bb9; /* 悬停时边框颜色更深 */
}

/* 点击时的效果 */
QLabel:pressed {
    background-color: #c3dafb; /* 按下时背景更浅 */
    color: #1a7bb9; /* 按下时文本颜色深 */
    border-color: #1a7bb9; /* 按下时边框深色 */
}

/* 禁用状态 */
QLabel:disabled {
    color: #b0c4de; /* 禁用状态的文本颜色 */
    background-color: #f0f4f8; /* 禁用状态的背景色 */
    border: 1px solid #d1dbe6; /* 禁用状态的边框 */
}

/* 聚焦效果 */
QLabel:focus {
    border: 2px solid #0078d4; /* 聚焦时边框颜色变为深蓝色 */
}

/* 如果 QLabel 设置了图片背景 */
QLabel[hasImage=&quot;true&quot;] {
    background: url(:/images/blue_bg.png) no-repeat center center;
    background-size: cover;
    color: #ffffff; /* 背景图片时，文本颜色为白色 */
    padding: 0;
}

/* 自定义标签图标 */
QLabel[icon]:not([hasImage=&quot;true&quot;]) {
    padding-left: 35px;
    background: transparent;
}

QLabel[icon]:not([hasImage=&quot;true&quot;])::before {
    content: url(:/images/icon_blue.png); /* 你可以自定义图标路径 */
    position: absolute;
    left: 12px;
    top: 50%;
    transform: translateY(-50%);
}

/* 更细致的调整：避免字体过大或者过小影响对齐 */
QLabel {
    line-height: 1.4; /* 增加行高，使文本垂直居中 */
    text-align: center;
}

/* 调整带图标时的显示效果 */
QLabel[icon]:not([hasImage=&quot;true&quot;]) {
    padding-left: 45px; /* 增加左侧的空隙，以确保图标不被文本遮挡 */
}
</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_priority">
           <property name="styleSheet">
            <string notr="true">QLabel {
    /* 基础样式 */
    font-family: &quot;Segoe UI&quot;, &quot;Helvetica&quot;, sans-serif;
    font-size: 16px;
    font-weight: bold; /* 设置字体加粗 */
    color: #4a90e2; /* 主文本颜色 - 清新的蓝色 */
    padding: 12px;
    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0.2 #e6f1fd, stop:1 #a9c9f5); /* 清新的蓝色渐变背景 */
    border: 1px solid #4a90e2; /* 蓝色边框 */
    text-align: center;
    min-width: 160px;
    min-height: 40px;
    max-width: 300px;
    max-height: 80px;
}

/* 鼠标悬停效果 */
QLabel:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0.2 #d2e8fb, stop:1 #82aef2); /* 悬停时的渐变 */
    color: #1a7bb9; /* 悬停时文本颜色更深 */
    border: 1px solid #1a7bb9; /* 悬停时边框颜色更深 */
}

/* 点击时的效果 */
QLabel:pressed {
    background-color: #c3dafb; /* 按下时背景更浅 */
    color: #1a7bb9; /* 按下时文本颜色深 */
    border-color: #1a7bb9; /* 按下时边框深色 */
}

/* 禁用状态 */
QLabel:disabled {
    color: #b0c4de; /* 禁用状态的文本颜色 */
    background-color: #f0f4f8; /* 禁用状态的背景色 */
    border: 1px solid #d1dbe6; /* 禁用状态的边框 */
}

/* 聚焦效果 */
QLabel:focus {
    border: 2px solid #0078d4; /* 聚焦时边框颜色变为深蓝色 */
}

/* 如果 QLabel 设置了图片背景 */
QLabel[hasImage=&quot;true&quot;] {
    background: url(:/images/blue_bg.png) no-repeat center center;
    background-size: cover;
    color: #ffffff; /* 背景图片时，文本颜色为白色 */
    padding: 0;
}

/* 自定义标签图标 */
QLabel[icon]:not([hasImage=&quot;true&quot;]) {
    padding-left: 35px;
    background: transparent;
}

QLabel[icon]:not([hasImage=&quot;true&quot;])::before {
    content: url(:/images/icon_blue.png); /* 你可以自定义图标路径 */
    position: absolute;
    left: 12px;
    top: 50%;
    transform: translateY(-50%);
}

/* 更细致的调整：避免字体过大或者过小影响对齐 */
QLabel {
    line-height: 1.4; /* 增加行高，使文本垂直居中 */
    text-align: center;
}

/* 调整带图标时的显示效果 */
QLabel[icon]:not([hasImage=&quot;true&quot;]) {
    padding-left: 45px; /* 增加左侧的空隙，以确保图标不被文本遮挡 */
}
</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0">
     <item>
      <widget class="QChartView" name="pieChartView">
       <property name="renderHints">
        <set>QPainter::RenderHint::Antialiasing|QPainter::RenderHint::TextAntialiasing</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="description_edit">
       <property name="styleSheet">
        <string notr="true">QTextEdit {
    background-color: #f4f7fa;  /* 背景色 */
    border: 2px solid #b0c4de;  /* 边框颜色 */
    border-radius: 12px;        /* 圆角 */
    padding: 15px 20px;         /* 内边距 */
    font-family: &quot;Helvetica Neue&quot;, &quot;Arial&quot;, sans-serif;  /* 字体 */
    font-size: 15px;            /* 字号 */
    color: #333;                /* 字体颜色 */
    line-height: 1.6;           /* 行高 */
    text-align: justify;        /* 文字对齐 */
    background: linear-gradient(135deg, #e0f7fa, #ffffff); /* 渐变背景 */
}

QTextEdit:focus {
    border-color: #66afe9;      /* 获取焦点时的边框颜色 */
    background-color: #ffffff;  /* 获取焦点时的背景色 */
}

QTextEdit:hover {
    border-color: #5b9bd5;      /* 鼠标悬停时的边框颜色 */
    background-color: #f0f8ff;  /* 鼠标悬停时的背景色 */
}

QTextEdit:disabled {
    background-color: #e9ecef;  /* 禁用时的背景色 */
    color: #888;                /* 禁用时字体颜色 */
    border-color: #d1d1d1;      /* 禁用时的边框颜色 */
}

QTextEdit::placeholder {
    color: #a9a9a9;             /* 占位符颜色 */
    font-style: italic;         /* 占位符字体样式 */
    font-size: 14px;            /* 占位符字体大小 */
}

QTextEdit[readOnly=&quot;true&quot;] {
    background-color: #f4f4f4;  /* 只读模式背景色 */
    color: #777;                /* 只读模式字体颜色 */
    border: 2px solid #ddd;     /* 只读模式边框 */
}

QTextEdit:enabled {
    background-color: #ffffff;  /* 启用时的背景色 */
    color: #333;                /* 启用时的字体颜色 */
    border-color: #b0c4de;      /* 启用时的边框颜色 */
}

QTextEdit::cursor {
    background-color: #ff6347;  /* 光标颜色 */
    width: 2px;                 /* 光标宽度 */
}

QTextEdit::selection {
    background-color: rgba(102, 175, 233, 0.4); /* 选中文本的背景色 */
    color: #fff;                /* 选中文本的字体颜色 */
}

QTextEdit:focus::selection {
    background-color: rgba(91, 155, 213, 0.5);  /* 聚焦时选中文本的背景色 */
}

QTextEdit[readOnly=&quot;true&quot;]:hover {
    border-color: #d1d1d1;      /* 只读模式悬停时的边框颜色 */
}

QTextEdit:enabled:hover {
    background-color: #f9f9f9;  /* 启用时悬停时的背景色 */
    border-color: #8db9e1;      /* 启用时悬停时的边框颜色 */
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
